### PR TITLE
Dev update 54

### PR DESCRIPTION
**Pull request by sdbondi**
## Update 54
- Tari merge mining pool (nodejs-pool)

### CODE REVIEW

File: _updates/2021-04-21-update-54.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+If you are adventurous and would like to run your own merge mining pool on your local machine, or deploy it on a public server you can
+find some basic instructions to get you started in the [README.md](https://github.com/tari-project/nodejs-pool/blob/tari/README.md#tari-merge-mining).
+
+If one or two evenings of devops doesn't appeal to you, we'll be making our deployment of nodejs-pool available for anyone to use in 
+the coming days. You'll be able to connect your miner (typically xmrig) to the pool. Just provide concatenated (tari:monero) wallet
+addresses and earn both testnet coins. 
+
+For example, if your _Tari_ address is `c6037038cce673cf1c1e26e7b7609ffbbba297eae4caf3ce7123a949c7af0162` and your _moenro_ address is `581EfC8NL2yLZyavVrY2dGZNsqBzzcU5yE8iLrFwPLkQHxUckaWy8vjeLntrynGoBwLeKTBdgJ2rR66ZiqKogkhpS8DAMR3`
+your xmrig config will be as follows: 
+```json
```
Comment (sdbondi) [inline _updates/2021-04-21-update-54.md:24]: Does this work in jekyll?

Comment (delta1) [reply _updates/2021-04-21-update-54.md:24]: will take a look

File: _updates/2021-04-21-update-54.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Early support for Tari Merge Mining has been added to the [Tari fork of nodejs-pool](https://github.com/tari-project/nodejs-pool).
+
+If you are adventurous and would like to run your own merge mining pool on your local machine, or deploy it on a public server you can
+find some basic instructions to get you started in the [README.md](https://github.com/tari-project/nodejs-pool/blob/tari/README.md#tari-merge-mining).
+
+If one or two evenings of devops doesn't appeal to you, we'll be making our deployment of nodejs-pool available for anyone to use in 
+the coming days. You'll be able to connect your miner (typically xmrig) to the pool. Just provide concatenated (tari:monero) wallet
+addresses and earn both testnet coins. 
+
+For example, if your _Tari_ address is `c6037038cce673cf1c1e26e7b7609ffbbba297eae4caf3ce7123a949c7af0162` and your _moenro_ address is `581EfC8NL2yLZyavVrY2dGZNsqBzzcU5yE8iLrFwPLkQHxUckaWy8vjeLntrynGoBwLeKTBdgJ2rR66ZiqKogkhpS8DAMR3`
```
Comment (delta1) [inline _updates/2021-04-21-update-54.md]: s/moenro/monero/

File: _updates/2021-04-21-update-54.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+## Tari Merge Mining Pool 
+
+Early support for Tari Merge Mining has been added to the [Tari fork of nodejs-pool](https://github.com/tari-project/nodejs-pool).
+
+If you are adventurous and would like to run your own merge mining pool on your local machine, or deploy it on a public server you can
+find some basic instructions to get you started in the [README.md](https://github.com/tari-project/nodejs-pool/blob/tari/README.md#tari-merge-mining).
+
+If one or two evenings of devops doesn't appeal to you, we'll be making our deployment of nodejs-pool available for anyone to use in
```
Comment (delta1) [inline _updates/2021-04-21-update-54.md:18]: lmao

**Review by delta1 — COMMENTED** [review]
will render locally quick

**Comment by delta1** [comment]
json looks okay enough 

image

**Review by delta1 — APPROVED** [review]
LGTM

File: _updates/2021-04-21-update-54.md (hunk 1 above)
Comment (sdbondi) [reply _updates/2021-04-21-update-54.md:24]: Thanks :)

**Review by stringhandler — APPROVED** [review]

**Pull request merged**